p4sdk/images/entrypoint.sh

```diff
@@ -60,4 +61,5 @@ cp /opt/p4/p4-cp-nws/share/stratum/es2k/es2k_port_config.pb.txt /usr/share/strat
 
 # Start Infrap4d
 func_set_br_pipe &
+sleep 10
```

Comment:
there is no need for this sleep instruction

Reply:
Didn't we observe infrap4d fails to initialize if it starts too soon? The sleep in func_set_br_pipe() in independent of this sleep here as that will sleep on it's own thread. Basically we added 15s additional sleep for before both `p4rt-ctl` calls.

Reply:
Apart from `func_set_br_pipe()` which will run independent, the rest of the execution I believe is sequential - the infrap4d should command should not start before the previous ones finish running. There is a single p4rt-ctl call in this file which is the set-pipe. The sleep before set-pipe is needed because it allows for infrap4d to start.